samples/web/content/apprtc/js/infobox.js

```diff
@@ -141,15 +144,17 @@ InfoBox.prototype.buildStatsSection_ = function() {
   var captureStart = extractStatAsInt(this.stats_, 'ssrc',
       'googCaptureStartNtpTimeMs');
   var e2eDelay = computeE2EDelay(captureStart, this.remoteVideo_.currentTime);
-  if (this.callSetupDelay_ !== null) {
+  if (this.endTime_ !== null) {
+    contents += this.buildLine_('Call time',
+        this.formatInterval_(window.performance.now() - this.endTime_));
```

Comment:
The timer does not get stopped when the call ended, so this will be wrong when the info div is up but the call has hangup.